### DISCUSSION TITLE
Entity sensor.time not found

### DISCUSSION THREAD

**Discussion by dhruvpandeysm** [shannonhochkins/ha-component-kit#34] [Q&A] [answered]
When trying to use any card component, it gives following error on UI
```
ERROR
Entity sensor.time not found
Wt@http://localhost:3000/static/js/bundle.js:67789:15
De@http://localhost:3000/static/js/bundle.js:68042:10
CA@http://localhost:3000/static/js/bundle.js:65175:63
renderWithHooks@http://localhost:3000/static/js/bundle.js:42652:31
mountIndeterminateComponent@http://localhost:3000/static/js/bundle.js:45938:17
beginWork@http://localhost:3000/static/js/bundle.js:47234:20
callCallback@http://localhost:3000/static/js/bundle.js:32244:18
invokeGuardedCallbackDev@http://localhost:3000/static/js/bundle.js:32288:20
invokeGuardedCallback@http://localhost:3000/static/js/bundle.js:32345:35
beginWork$1@http://localhost:3000/static/js/bundle.js:52219:32
performUnitOfWork@http://localhost:3000/static/js/bundle.js:51466:16
workLoopSync@http://localhost:3000/static/js/bundle.js:51389:26
renderRootSync@http://localhost:3000/static/js/bundle.js:51362:11
performConcurrentWorkOnRoot@http://localhost:3000/static/js/bundle.js:50756:78
workLoop@http://localhost:3000/static/js/bundle.js:60510:46
flushWork@http://localhost:3000/static/js/bundle.js:60488:18
performWorkUntilDeadline@http://localhost:3000/static/js/bundle.js:60725:25
```

**Comment by shannonhochkins** [comment]
> When trying to use any card component, it gives following error on UI
> ```
> ERROR
> Entity sensor.time not found
> Wt@http://localhost:3000/static/js/bundle.js:67789:15
> De@http://localhost:3000/static/js/bundle.js:68042:10
> CA@http://localhost:3000/static/js/bundle.js:65175:63
> renderWithHooks@http://localhost:3000/static/js/bundle.js:42652:31
> mountIndeterminateComponent@http://localhost:3000/static/js/bundle.js:45938:17
> beginWork@http://localhost:3000/static/js/bundle.js:47234:20
> callCallback@http://localhost:3000/static/js/bundle.js:32244:18
> invokeGuardedCallbackDev@http://localhost:3000/static/js/bundle.js:32288:20
> invokeGuardedCallback@http://localhost:3000/static/js/bundle.js:32345:35
> beginWork$1@http://localhost:3000/static/js/bundle.js:52219:32
> performUnitOfWork@http://localhost:3000/static/js/bundle.js:51466:16
> workLoopSync@http://localhost:3000/static/js/bundle.js:51389:26
> renderRootSync@http://localhost:3000/static/js/bundle.js:51362:11
> performConcurrentWorkOnRoot@http://localhost:3000/static/js/bundle.js:50756:78
> workLoop@http://localhost:3000/static/js/bundle.js:60510:46
> flushWork@http://localhost:3000/static/js/bundle.js:60488:18
> performWorkUntilDeadline@http://localhost:3000/static/js/bundle.js:60725:25
> ```
> 

Hey! Thanks for reporting the issue, which version are you using? I'll have this sorted in a few hours if it's on the latest versions :)

**Reply by shannonhochkins** [reply]
Also, do you have sensor.time in your home assistant instance under the entities section? I was under the assumption that every home assistant setup had this sensor by default

**Reply by dhruvpandeysm** [reply]
Yes everything is latest.
```
"@hakit/components": "^1.0.20",
"@hakit/core": "^1.0.17",
Home Assistant 2023.8.4
```

No I cannot see the `sensor.time` entity in my Developer tools on HA

**Reply by shannonhochkins** [reply]
Thank you. And could you please just confirm if you have the time sensor? Search the entities section please :)

[Screenshot_20230905_185925_Home Assistant.jpg](https://github.com/shannonhochkins/ha-component-kit/assets/5105002/cdaf33b7-f4ff-4a5d-8fc8-0bd8d9149a3d)

**Reply by dhruvpandeysm** [reply]
No I cannot see the `sensor.time` entity in my Developer tools on HA

**Reply by shannonhochkins** [reply, marked as answer]
Interesting that you're the first person to raise this, after a quick google search, if you add the following to your configuration.yaml

```
sensor:
  - platform: time_date            
    display_options:
       - 'time' 
       - 'date'
```

Apologies for the formatting I'm on my phone, however once you restart it should all be okay!

Now that I know they're not always available I can make provisions for this going forward, however this should get you out of trouble for now!
